File: LogicadeProgramacao/Estrutura de Decisão/2. Java/Operacoes.java
import java.util.Scanner;

public class Operacoes {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        double num1 = input.nextDouble();

        System.out.print("Digite o segundo número: ");
        double num2 = input.nextDouble();

        System.out.print("Qual operação deseja realizar? (+, -, *, /): ");
        char operacao = input.next().charAt(0);

        double resultado = 0;

        switch (operacao) {
            case '+':
                resultado = num1 + num2;
                System.out.printf("O resultado de %.2f + %.2f é %.2f\n", num1, num2, resultado);
                break;

            case '-':
                resultado = num1 - num2;
                System.out.printf("O resultado de %.2f - %.2f é %.2f\n", num1, num2, resultado);
                break;

            case '*':
                resultado = num1 * num2;
                System.out.printf("O resultado de %.2f * %.2f é %.2f\n", num1, num2, resultado);
                break;

            case '/':
                if (num2 != 0) {
                    resultado = num1 / num2;
                    System.out.printf("O resultado de %.2f / %.2f é %.2f\n", num1, num2, resultado);
                } else {
                    System.out.println("Não é possível dividir por zero.");
                }
                break;

            default:
                System.out.println("Operação inválida.");
                break;
        }

        if (resultado % 2 == 0) {
            System.out.println("O resultado é par.");
        } else {
            System.out.println("O resultado é ímpar.");
        }

        if (resultado >= 0) {
            System.out.println("O resultado é positivo.");
        } else {
            System.out.println("O resultado é negativo.");
        }

        if (resultado == Math.round(resultado)) {
            System.out.println("O resultado é inteiro.");
        } else {
            System.out.println("O resultado é decimal.");
        }

        input.close();
    }
}

File: LogicadeProgramacao/Estrutura de Decisão/3. Python/Operacoes.py
num1 = float(input("Digite o primeiro número: "))
num2 = float(input("Digite o segundo número: "))

operacao = input("Qual operação deseja realizar? (+, -, *, /): ")

resultado = 0

if operacao == '+':
    resultado = num1 + num2
    print(f"O resultado de {num1:.2f} + {num2:.2f} é {resultado:.2f}")

elif operacao == '-':
    resultado = num1 - num2
    print(f"O resultado de {num1:.2f} - {num2:.2f} é {resultado:.2f}")

elif operacao == '*':
    resultado = num1 * num2
    print(f"O resultado de {num1:.2f} * {num2:.2f} é {resultado:.2f}")

elif operacao == '/':
    if num2 != 0:
        resultado = num1 / num2
        print(f"O resultado de {num1:.2f} / {num2:.2f} é {resultado:.2f}")
    else:
        print("Não é possível dividir por zero.")

else:
    print("Operação inválida.")

if resultado % 2 == 0:
    print("O resultado é par.")
else:
    print("O resultado é ímpar.")

if resultado >= 0:
    print("O resultado é positivo.")
else:
    print("O resultado é negativo.")

if resultado.is_integer():
    print("O resultado é inteiro.")
else:
    print("O resultado é decimal.")
